lib/public_api.ts

```diff
@@ -1,22 +1,22 @@
-export * from './src/accordion/accordion.module';
-export * from './src/breadcrumbs/breadcrumbs.module';
-export * from './src/app-switcher/index';
-export * from './src/button/button.module';
-export * from './src/checkbox/checkbox.module';
-export * from './src/chip/chip.module';
-export * from './src/drawer/drawer.module';
-export * from './src/icon/icon.module';
-export * from './src/input/input.module';
-export * from './src/label/label.module';
-export * from './src/list/list.module';
-export * from './src/modal/index';
-export * from './src/navbar/navbar.module';
-export * from './src/pipes/pipes.module';
-export * from './src/popover/popover.module';
-export * from './src/progress-indicators/progress-indicators.module';
-export * from './src/radio-button/radio-button.module';
-export * from './src/select/select.module';
-export * from './src/subnav/subnav.module';
-export * from './src/tabs/tabs.module';
-export * from './src/tile/tile.module';
-export * from './src/typeform-survey/typeform-survey.module';
+export { AccordionModule } from './src/accordion/accordion.module';
+export { BreadcrumbsModule } from './src/breadcrumbs/breadcrumbs.module';
+export { AppSwitcherModule } from './src/app-switcher/app-switcher.module';
```

Comment:
This exported the `app-switcher.service.ts` and the `app-switcher-interfaces.ts` which wasn't included in the module.  I am not sure they will be exported, can you verify this or include them to be excluded somehow?

Reply:
Good point.  I'll adjust this.